components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -46,6 +47,13 @@
     String applicationSPName = request.getParameter("spName");
     session.setAttribute("application-sp-name", applicationSPName);
 
+    String encryptionAlgorithm = OAuthServerConfiguration.getInstance().getIdTokenEncryptionAlgorithm();
```

Comment:
Here in UI side we are directly calling server side classes (OAuthServerConfiguration).
In design wise I think iti is not right.
Is it possible to use UI client and call the server side classes?

Reply:
I have changed the approach. Please review. :)